### ISSUE TITLE
Bug: Misreported location for `!important` when preceding comment has emoji

### ISSUE THREAD

**Issue by sethamus** [eslint/css#284]
### Environment

ESLint version: 9.37.0
@eslint/css version: 0.11.1
Node version: v22.20.0
npm version: 
Operating System:

### Which language are you using?

stylesheet

### What did you do?

```css
/* eslint css/no-important: "error" */

a { color: red /* comment 👍🚀 */ !important; }
```

### What did you expect to happen?

The reported location for `css/no-important` should point exactly at the `!important` token, regardless of any preceding comment content, including emoji.

### What actually happened?

The reported location is shifted when the comment immediately preceding `!important` contains Unicode surrogate pairs (e.g., emoji like 👍 or 🚀).

### Link to Minimal Reproducible Example

See code above

### Participation

- [x] I am willing to submit a pull request for this issue.
